src/storage/src/hummock/mod.rs

```diff
@@ -201,10 +206,24 @@ impl HummockStorage {
             local_version_manager.get_pinned_version(),
         )));
 
+        let (version_update_notifier_tx, seal_epoch) = {
+            let basic_max_committed_epoch = local_version_manager
+                .get_pinned_version()
+                .max_committed_epoch();
+            let (version_update_notifier_tx, _rx) =
+                tokio::sync::watch::channel(basic_max_committed_epoch);
+
+            (
+                Arc::new(version_update_notifier_tx),
+                Arc::new(AtomicU64::new(0)),
```

Comment:
The initial sealed epoch should be the MCE?

Reply:
fix it